kratos/python/add_matrix_to_python.cpp

```diff
@@ -89,6 +90,26 @@ namespace Python
 
           return matrix;
         }));
+
+        matrix_binder.def(py::init([](const py::list& input){
+          std::size_t num_rows = input.size();
+          if( num_rows == 0 || ((num_rows == 1) && ((Vector(py::cast<Vector>(input[0]))).size() == 0)) )
+            return DenseMatrix<double>(0,0);
+          else{
+            std::size_t num_cols = (Vector(py::cast<Vector>(input[0]))).size();
+            DenseMatrix<double>matrix = DenseMatrix<double>(num_rows, num_cols);
+            for(std::size_t i=0; i<num_rows; i++){
+              auto v_row = Vector(py::cast<Vector>(input[i]));
+              KRATOS_ERROR_IF( v_row.size() != num_cols ) << "Wrong size of a row\n";
+              for(std::size_t j=0; j<num_cols; j++){
+                  matrix(i,j) = v_row[j];
+              }
```

Comment:
Why do you use a `Vector` in the middle?

Why not directly filling the matrix?
I think this should be possible

Reply:
Do you mean like this 
`matrix(i,j) = py::cast<double>(input[i][j]); ` OR
`matrix(i,j) = input[i][j]` OR
are not allowed

Reply:
`matrix(i,j) = py::cast<double>((py::cast<py::list>(input[i]))[j]);`
Worked out !!!!

Reply:
why do you even need the cast? It is already a `py::list` no?

Reply:
Yes, it is py::list, but not able to do `py::cast<double>(input[i][j])`.
> error: no viable overloaded operator[] for type 'detail::list_accessor' (aka 'accessor<accessor_policies::list_item>')
>                   matrix(i,j) = py::cast<double>(input[i][j])